apps/frontend/src/components/PreReqCourses.tsx

```diff
@@ -0,0 +1,134 @@
+import React from "react";
+import { useFetchCourseInfo } from "~/app/api/course";
+
+interface TreeNode {
```

Comment:
I notice use define TreeNode in a few files, would be good to put one standard definition in apps/types